frontend/pages/DashboardPage/components/MdmSolutionModal/MdmSolutionModal.tsx

```diff
@@ -43,7 +41,7 @@ const MdmSolutionsModal = ({
   return (
     <Modal
       className={baseClass}
-      title={mdmSolutions[0].name ?? "Mdm solution"}
+      title={mdmSolutions[0].name ?? "Unknown mdm solution"}
```

Comment:
Nit: Should "MDM" be capitalized here?

Reply:
yes. Also i've changed it so that empty string also show the default title.